src/components/settings-modal/settings-modal.js

```diff
@@ -23,6 +23,8 @@ const SettingsModal = ({
   onTogglePrettyName,
   visible,
   flags,
+  isOutdated,
+  latest,
```

Comment:
Maybe `latest` can be a little more specific? It's pretty vague as is. Like `latestVizVersion` or something? 

Sort here if possible.